### ISSUE TITLE
[v2] Accessibility not found for element inside bottom sheet (iOS only)

### ISSUE THREAD

**Issue by moxspoy** [gorhom/react-native-scroll-bottom-sheet#712]
# Bug

Hi! i am working to automate test component that using BottomSheet. The problem is Appium (my test framework) can not get element inside the bottom sheet on iOS device. However, it works on Android device.
My expectation is i can get the element inside the bottom sheet so that i can take action such as press on the button etc.
The actual condition is, i only get one accessibility label "Bottom Sheet" that covering all area in the screen.

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | 2.4.0   |
| react-native                    | 0.61.5   |
| react-native-reanimated         | 1.7.0   |
| react-native-gesture-handler    | 1.3.0   |

## Steps To Reproduce

1. Open Appium & use inspect element
2. Open the bottom sheet in the app
3. select the element inside the bottom sheet (e.g text component)
4. The accessibility label for the text is not found

Describe what you expected to happen:

1. The text inside the bottom sheet which i selected above can be found

## Reproducible sample code

```js
return (
        <View style={Style.container}>
            <Button text={'TOGGLE'} onPress={onPress} />
            <BottomSheet
                ref={bottomSheetRef}
                snapPoints={snapPoints}
                backdropComponent={BottomSheetBackdrop}>
               <View style={{
                   backgroundColor: Color.white,
                   padding: 30,
               }}>
                   <Text accessibilityLabel="this_label">This</Text>
                   <Text>is</Text>
                   <Text>content</Text>
                   <Text>that is not accessible</Text>
               </View>
            </BottomSheet>
        </View>
    );
```

![Screen Shot 2021-10-21 at 10 42 51](https://user-images.githubusercontent.com/17137806/138207896-ee842dbb-faf9-4400-bd30-ec06cccd5db0.png)

**Comment by JoNesVai** [comment]
I have the same problem. Is there any solution for this problem or some fix in near future scope?

**Comment by turker0** [comment]
Having same issue

**Comment by JoNesVai** [comment]
@oguzturker8 Update to the newest version worked for me :)

**Comment by moxspoy** [comment]
i can confirm that the issue has been solved in the latest release :)

**Comment by turker0** [comment]
Thanks @JoNesVai @moxspoy, I'll be trying the newest version

**Comment by turker0** [comment]
@JoNesVai @moxspoy Hi again, updated to ^2.4.1 but it seems not working? Which version do you use?

**Comment by moxspoy** [comment]
have you try using bottom sheet in simple RN project? you can inspect the element using inspector tools like appium

im using bottom sheet 4.5.1 and react native reanimated v2.4.1

**Comment by turker0** [comment]
I thought it was solved on bottom sheet v2 version 2.4.1. Unfortunately i couldn't upgrade rn version to +0.63.

**Comment by moxspoy** [comment]
i suggest you to upgrade the version to have the latest updates both in animation or anything.
